build.gradle

```diff
@@ -27,6 +27,17 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven {
```

Comment:
We don't need this node to get merged into `main` branch this is just for testing purposes. This is what is making @ancirja-m PR to fail: https://github.com/microsoft/surface-duo-sdk-samples-kotlin/runs/1922441837?check_suite_focus=true (it fails because the build system doesn't have a secret with USERNAME and PASSWORD)